packages/input-option/src/InputOption/InputOption.types.ts

```diff
@@ -26,9 +44,9 @@ export interface BaseInputOptionProps {
   highlighted?: boolean;
```

Comment:
Can we update the above `TERMINOLOGY` section

Reply:
yep!